src/rx5808-pro-diversity/settings.h

```diff
@@ -83,6 +89,13 @@ SOFTWARE.
     #define DIVERSITY_MAX_CHECKS 5
 #endif
 
+#ifdef FAST_DIVERSITY_SWITCHING
+    // see https://www.arduino.cc/en/Reference/PortManipulation
+    // PORTC means analog ports (PORTD is arduino 0-7, PORTB is 8-13)
+    // PC0 is the current value of A0 (receiver A) PC1 is value of A1 (receiver B)
+    #define TOGGLE_RECEIVER PORTC ^= (1 << PC0)|(1 << PC1);
```

Comment:
The result of `(1 << PC0)|(1 << PC1)` is always binary 3 (11) in this use case where there is one active and one inactive.
Is there a particular reason to do it this way?
I would suggest to pre calculate this and use binary 3 `B00000011`.
Now if someone wanted to switch A0 and A2 they can write `B00000101`.